src/main/java/mate/academy/dao/BookDao.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.dao;
+
+import mate.academy.model.Book;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface BookDao {
+    Book create(Book book);
+    Optional<Book> findById(Long id);
+    List<Book> findAll();
+    Book update(Book book);
+    boolean deleteById(Long id);
```

Comment:
don't forget to put spaces between methods 
```suggestion
    Book create(Book book);
    
    Optional<Book> findById(Long id);
    
    List<Book> findAll();
    
    Book update(Book book);
    
    boolean deleteById(Long id);
```